### PR TITLE
Books; small corrections to title and formatting

### PR DESCRIPTION
**Pull request by MattDodsonEnglish**
These fixes are very minor. 

However, while looking at that list, I also found two other fixes to propose:

* Perhaps the titles should be italicized
* The list has a good number of items. If it were broken up into a few topic categories, it might be easier for readers to process.

If the maintainers think those changes would be worthwhile, I'd be happy to make them myself sometime in the near future.

### CODE REVIEW

**Review by plaindocs — APPROVED** [review]
Thanks for the fixes!

**Comment by plaindocs** [comment]
I'd be happy to merge both readability fixes you propose, might be worth using similar categories to the[ docs guide](https://www.writethedocs.org/guide/) if that works.

**Comment by MattDodsonEnglish** [comment]
I just looked at the "Docs Guide" list. And while it definitely provides excellent coverage of the major topics in tech writing,
I think it might be hard to fit the books from that list into such specific categories. Grouping them like that would be especially hard for me, as I haven't even read most of those books :-) .

It looks like each book on the current list fits comfortably into these broad categories:

* Books specifically about technical writing
* Books about writing web content
* Books about design and information architecture
* Books about the craft of writing in general

Maybe for now I could just split it into 3 or 4 parts (I'm not sure yet whether _web content_ should be separated from _tech writing_). Later, when more items are added to the list (or when someone who is more well-read than me comes along),  Read the Docs could consider regrouping the list into more specific sets.

What do you think?

**Comment by plaindocs** [comment]
That all sounds sensible to me. :+1:

**Comment by MattDodsonEnglish** [comment]
Now with commit aa8b5527fd658200ffa5b75ec9006441ea9d28ca , I've also arranged the books into four topics.

To avoid nested formatting, I decided against italicizing the titles (apparently it'd be tricky in RST).

If it looks okay, I think that's all from me. Thanks for your collaboration!

**Comment by plaindocs** [comment]
Thanks a ton @MattDodsonEnglish, I'll get this merged in.

File: docs/books.rst [hunk 1]
```diff
@@ -3,33 +3,48 @@ Relevant Books
...
 * `It Was the Best of Sentences, It Was the Worst of Sentences <https://www.penguinrandomhouse.com/books/198856/it-was-the-best-of-sentences-it-was-the-worst-of-sentences-by-june-casagrande/>`_ , by June Casagrande
-* `Letting Go of the Words <https://www.goodreads.com/book/show/1135441.Letting_Go_of_the_Words>`_, by Janice (Ginny) Redish - **Read in our book club**
-* `Living Documentation <https://www.goodreads.com/book/show/26865837-living-documentation-by-design-with-domain-driven-design>`_, by Cyrille Martraire
-* `Managing Writers: A Real World Guide to Managing Technical Documentation <https://www.goodreads.com/book/show/6322901-managing-writers>`_, by Richard L. Hamilton - **Read in our book club**
-* `Modern Technical Writing <https://www.goodreads.com/book/show/28433138-modern-technical-writing>`_, by Andrew Etter
 * `Nicely Said <http://www.nicelysaid.co/>`_, by Nicole Fenton and Kate Kiefer Lee
 * `On Writing <https://stephenking.com/works/nonfiction/on-writing-a-memoir-of-the-craft.html>`_, by Stephen King
 * `The Organised Writer <http://organised-writer.com/>`_, by Antony Johnston - **Upcoming book club read**
```
Comment (plaindocs) [inline docs/books.rst]:
```suggestion
* `The Organised Writer <http://organised-writer.com/>`_, by Antony Johnston - **Upcoming book club read August 2021**
```

**Pull request merged**